.github/workflows/spec-update-cron.yaml

```diff
@@ -11,17 +11,17 @@ on:
 
 jobs:
   spec-update-ai-api:
-    uses: SAP/ai-sdk-js/.github/workflows/spec-update.yaml@main
+    uses: SAP/ai-sdk-js/.github/workflows/spec-update.yaml@v1-main
     with:
       file: ai-api
     secrets: inherit
   spec-update-document-grounding:
-    uses: SAP/ai-sdk-js/.github/workflows/spec-update.yaml@main
+    uses: SAP/ai-sdk-js/.github/workflows/spec-update.yaml@v1-main
     with:
       file: document-grounding
     secrets: inherit
   spec-update-prompt-registry:
-    uses: SAP/ai-sdk-js/.github/workflows/spec-update.yaml@main
+    uses: SAP/ai-sdk-js/.github/workflows/spec-update.yaml@v1-main
```

Comment:
(Read my other comments below first)

For this cron job specifically, since we should only run spec update for v2, and orchestration use `main` to run the generator, I would not use `v1-main` here but `main` directly, and only update that one in main.

`spec-update.yaml` on v1 should not be used.